src/annotator/guest.js

```diff
@@ -361,6 +356,15 @@ export default class Guest {
       }
     );
 
+    this._hostRPC.on(
+      'selectAnnotations',
+      /**
+       * @param {string[]} tags
+       * @param {boolean} toggle
+       */
+      (tags, toggle) => this.selectAnnotations(tags, toggle)
```

Comment:
There is a slightly problem with this approach. _Every_ `Guest` will receive the RPC event and _every_ `Guest` will ask the `Sidebar` to show or toggle on the annotations.